packages/ui/theme/foundations/fonts.ts

```diff
@@ -1,8 +1,8 @@
 import type { ThemeOverride } from '@chakra-ui/react';
-import primitives from '@core-ds/primitives';
+// import primitives from '@core-ds/primitives';
 
 export const fonts: ThemeOverride['fonts'] = {
-   body: primitives.fontFamily.lato,
-   heading: primitives.fontFamily.lato,
-   mono: primitives.fontFamily.monoSystem,
+   body: `-apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"`,
+   heading: `-apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"`,
```

Comment:
GitHub has the Emoji fonts, but it seems a little weird since we don't really use emoji. I guess the idea is that it can't hurt, and if there do end up being emoji somewhere they'll look like the user's system emoji?